conf/application.conf

```diff
@@ -158,8 +158,10 @@ akka.http.parsing.max-uri-length = 16k
 # Feature flags
 # This flag is used to enable the capture of the "referer" header if the referrerUrl is not passed via the query string
 useRefererHeader = true
-# This flag is used to disable the partials endpoints
+# This flag is used to disable the non-Ajax partials endpoints
 disablePartials = false
+# This flag is used to disable the non-Ajax partial endpoint
```

Comment:
```suggestion
# This flag is used to disable the Ajax partial endpoints
```

Reply:
@ellamdav In fact this is only used to disable to one endpoint, but happy to amend if you like?

Reply:
ah, it was more the `non-` prefix, but I also just made the plurality match the variable name